pkg/serviceaccounts/controllers/create_dockercfg_secrets.go

```diff
@@ -47,6 +51,12 @@ type DockercfgControllerOptions struct {
 	DockerURLsIntialized chan struct{}
 }
 
+// pendingTokenStore stores the names of the tokens created by this controller
+type pendingTokenStore struct {
```

Comment:
This structure suggests to me that this won't function properly if multipel controllers activate.